packages/lib/models/Setting.ts

```diff
@@ -1546,6 +1554,27 @@ class Setting extends BaseModel {
 				public: false,
 			},
 
+			installedDefaultPlugins: {
+				value: [],
+				type: SettingItemType.Array,
+				public: false,
+				storage: SettingStorage.File,
+			},
+
+			setInitialDefaultPluginsSettings: {
+				value: [],
+				type: SettingItemType.Array,
+				public: false,
+				storage: SettingStorage.File,
+			},
+
+			preInstalledDefaultPlugins: {
+				value: '',
+				type: SettingItemType.Object,
+				public: false,
+				storage: SettingStorage.File,
+			},
+
```

Comment:
Please remove these in favour of using the `installedDefaultPlugins` field alone.